### PR TITLE
[HELP] Update Romanian (ro-RO) translation

### PR DESCRIPTION
**Pull request by AndreiMiloiu**
Attendum to PR https://github.com/reactos/reactos/pull/5908

### CODE REVIEW

**Comment by AndreiMiloiu** [comment]
Cc @reactos/lang-romanian

File: base/applications/cmdutils/help/lang/ro-RO.rc [hunk 1]
```diff
@@ -1,8 +1,9 @@
 /*
- * FILE:       base/applications/cmdutils/help/lang/ro-RO.rc
- *             ReactOS Project (https://reactos.org)
- * AUTHOR:     Magnus Olsen, 2005
- * TRANSLATOR: Ștefan Fulea (stefan dot fulea at mail dot com)
+ * PROJECT:     ReactOS DX Diagnostic
```
Comment (JoachimHenze) [inline base/applications/cmdutils/help/lang/ro-RO.rc:2]: This has absolutely nothing to do with "ReactOS DX Diagnostic". A good example, why it is a foolish idea to fill all of our rcs with obsolete information by duplicating wrong data into 36 or more rc files.

Comment (AndreiMiloiu) [reply base/applications/cmdutils/help/lang/ro-RO.rc:2]: Sorry, I copied the header on my computer and I forgot to change the name when I pasted into the translated file from PR. I will modify it right now.

Comment (JoachimHenze) [reply base/applications/cmdutils/help/lang/ro-RO.rc]: No problem. It is obvious how copy-paste-issues do evolve. Clever programmers do apply workflows therefore that do minimize the need for copy-n-paste. Again that comment is not so much targetting you (just following orders you received), but to the "ordererers".

**Pull request merged**